### ISSUE TITLE
go-talks.appspot.com could use "Refresh now" functionality

### ISSUE THREAD

**Issue by AlekSi** [golang/gddo#281]
Like godoc.org

**Comment by garyburd** [comment]
Talksapp automatically refreshes the presentation once per hour.

**Comment by AlekSi** [comment]
Yes, but sometimes it's not often enough. Context: https://twitter.com/paaleksey/status/615087399115333633

**Comment by adg** [comment]
cc @campoy

**Comment by mdlayher** [comment]
I've submitted a proposed solution to this issue here: https://go-review.googlesource.com/c/29651/.

@campoy and @garyburd  , I'd appreciate a review!
